sktime/forecasting/base/_base.py

```diff
@@ -616,7 +617,8 @@ def predict_quantiles(self, fh=None, X=None, alpha=None):
         # input checks and conversions
 
         # check fh and coerce to ForecastingHorizon, if not already passed in fit
-        fh = self._check_fh(fh)
+        cutoff = get_cutoff(self._y, self.cutoff, return_index=True)
```

Comment:
can you explain why you are recomputing the cutoff, can we not get `self._cutoff`?

Reply:
I am not sure if `self._cutoff` is updated before `_chech_fh`. In some functions like `update_predict_single`, `_check_cutoff` is actully called before `_update_y_X`. I moved `_check_cutoff` and removed passing cutoff now.